BlockParty.py

```diff
@@ -513,6 +524,7 @@ def init_game(self):
         self.clear_glass()
         self.complete_update = True
         self.glass_update = True
+        self.canSpeedUp = True
```

Comment:
Could you stick to the naming convention in the repo? You've used camel case naming and that's not what's used in the repo.

Reply:
Oops, updated :D